hasjob/templates/layout.html.jinja2

```diff
@@ -245,63 +255,83 @@
 {% endblock %}
 
 {% block basecontent -%}
-  {%- if header_campaign %}<div id="header-campaign"></div>{% endif %}
+  {%- if header_campaign and not embed %}<div id="header-campaign"></div>{% endif %}
   <div id='main-content'>
   {% block pagecontent %}{% block content %}{% endblock %}{% endblock %}
   </div>
-  {%- if header_campaign %}{{ campaign_header(g, header_campaign) }}{% endif %}
+  {%- if header_campaign and not embed %}{{ campaign_header(g, header_campaign) }}{% endif %}
 {%- endblock %}
 
+{% block basefooter -%}
+  {%- if not embed %}
+    {{ super() }}
+  {% endif %}
+{% endblock %}
```

Comment:
Instead of this, make an `embedlayout.html.jinja2` and in `index.html.jinja2`, add an `if` condition to decide which template to extend.